here-naksha-lib-jbon/src/jsMain/kotlin/JsMapApi.kt

```diff
@@ -59,4 +59,10 @@ class JsMapApi : IMapApi {
     override fun iterator(map: IMap): IMapIterator {
         return JsMapInterator(map, keys(map))
     }
+
```

Comment:
We should not mix Object and Map usage. Actually, we're currently only using Object as underlying type for IMap, the reason is simply that the JSON.parse method returns only objects. Can we therefore stick to object?